Manifests/ManagedPreferencesApplications/com.microsoft.wdav.plist

```diff
@@ -656,40 +933,390 @@ Off: your device is not protected against this type of threat and nothing is log
+															<string>Permission level</string>
+															<key>pfm_type</key>
+															<string>array</string>
+														</dict>
+														<dict>
+															<key>pfm_name</key>
+															<string>serialNumbers</string>
+															<key>pfm_subkeys</key>
+															<array>
+																<dict>
+																	<key>pfm_name</key>
+																	<string>{{key}}</string>
+																	<key>pfm_title</key>
+																	<string>Serial Number</string>
+																	<key>pfm_type</key>
+																	<string>string</string>
+																</dict>
+																<dict>
+																	<key>pfm_name</key>
+																	<string>{{value}}</string>
+																	<key>pfm_subkeys</key>
+																	<array>
+																		<dict>
+																			<key>pfm_range_list</key>
+																			<array>
+																				<string>none</string>
+																				<string>read</string>
+																				<string>write</string>
+																				<string>execute</string>
+																			</array>
+																			<key>pfm_range_list_titles</key>
+																			<array>
+																				<string>None</string>
+																				<string>Read</string>
+																				<string>Write</string>
+																				<string>Execute</string>
+																			</array>
+																			<key>pfm_title</key>
+																			<string>Permissions</string>
+																			<key>pfm_type</key>
+																			<string>string</string>
+																		</dict>
+																	</array>
+																	<key>pfm_title</key>
+																	<string>Serial Number Details</string>
+																	<key>pfm_type</key>
+																	<string>array</string>
+																</dict>
+															</array>
+															<key>pfm_title</key>
+															<string>Serial Numbers</string>
+															<key>pfm_type</key>
+															<string>dictionary</string>
+														</dict>
+													</array>
+													<key>pfm_title</key>
+													<string>Product Details</string>
+													<key>pfm_type</key>
+													<string>dictionary</string>
+												</dict>
+											</array>
+											<key>pfm_title</key>
+											<string>Products</string>
+											<key>pfm_type</key>
+											<string>dictionary</string>
+										</dict>
+									</array>
+									<key>pfm_title</key>
+									<string>Vendor Details</string>
+									<key>pfm_type</key>
+									<string>dictionary</string>
+								</dict>
+							</array>
+							<key>pfm_title</key>
+							<string>Vendors</string>
+							<key>pfm_type</key>
+							<string>dictionary</string>
+						</dict>
+					</array>
+					<key>pfm_title</key>
+					<string>Allow or block removable devices</string>
+					<key>pfm_type</key>
+					<string>dictionary</string>
+				</dict>
+			</array>
+			<key>pfm_title</key>
+			<string>Device control policy</string>
```

Comment:
nit: can you also pls bump the `pfm_version` at the bottom of the manifest pls?

Reply:
@apizz how does that look now?